crates/bevy_camera_controller/src/free_cam.rs

```diff
@@ -1,37 +1,67 @@
-//! A freecam-style camera controller plugin.
-//! To use in your own application:
-//! - Copy the code for the [`CameraControllerPlugin`] and add the plugin to your App.
-//! - Attach the [`CameraController`] component to an entity with a [`Camera3d`].
+//! A camera controller that allows the user to move freely around the scene.
 //!
-//! Unlike other examples, which demonstrate an application, this demonstrates a plugin library.
+//! Free cams are helpful for exploring large scenes, level editors and for debugging.
```

Comment:
These docs are gone now from docs.rs due to the module not being pub, FYI

Reply:
Ugh, that's so sad. Reverting.